src/resources/2017/Spacegate.ts

```diff
@@ -0,0 +1,155 @@
+import { cliExecute, Item, visitUrl } from "kolmafia";
+import { get } from "../../property";
+import { $item, $items } from "../../template-string";
+
+export function have(): boolean {
+  return get("spacegateAlways") || get("_spacegateToday");
+}
+
+export function checkStatus(): void {
+  visitUrl("place.php?whichplace=spacegate&action=sg_Terminal");
+}
+
+export function dialled(): boolean {
+  checkStatus();
+  return get("_spacegateCoordinates") !== "";
+}
+
+export function hazards() {
+  return get("_spacegateHazards");
+}
+
+export function planetName() {
+  return get("_spacegatePlanetName");
+}
+
+export function planetCoords() {
+  return get("_spacegateCoordinates");
+}
+
+export function plantLife() {
+  return get("_spacegatePlantLife");
+}
+
+export function animalLife() {
+  return get("_spacegateAnimalLife");
+}
+
+export function intelligentLife() {
+  return get("_spacegateIntelligentLife");
+}
+
+export function ruins() {
+  return get("_spacegateRuins");
+}
+
+export function spants() {
+  return get("_spacegateSpant");
+}
+
+export function murderBots() {
+  return get("_spacegateMurderbot");
+}
+
+export function hazardEquipment(hazards: string): Item | Item[] {
```

Comment:
I think returning just Item[], even if that array is sometimes a singleton, is probably more useful?

Reply:
Makes sense, wasn't sure if it allowed singleton $item arrays